xwiki-rendering-api/src/main/java/org/xwiki/rendering/block/ImageBlock.java

```diff
@@ -54,7 +62,7 @@
      */
     public ImageBlock(ResourceReference reference, boolean freestanding)
     {
-        this(reference, freestanding, Collections.<String, String>emptyMap());
+        this(reference, freestanding, null, Collections.emptyMap());
```

Comment:
`Map.of()` is also an option.

Reply:
We also have `Listener.EMPTY_PARAMETERS`, this change is mostly to remove the superfluous type parameters.